packages/documentation-component/src/core/plugins.ts

```diff
@@ -120,9 +120,9 @@ function runPlugins(sources: Sources, plugins?: Plugins): Sources {
     return sources;
   }
 
-  let processedSources: Sources = [];
+  let processedSources: Sources = sources;
```

Comment:
I only need shallow copy, because for in `data` object dev can pass object with circular referencing, and also with refs components works faster. On next iteration we can thinking about changing implementation.

Reply:
"next iteration" is "never", now is the time to change this